src/Config.js

```diff
@@ -7,6 +7,29 @@ class Config {
     return `https://${this.sinopiaDomainName}`
   }
 
+  static get sinopiaServerUrl() {
+    return process.env.REACT_APP_SINOPIA_SERVER_URL || 'http://localhost:8080'
+  }
+
+  static get resourceTemplateContainerPath() {
+    return 'ld4p'
```

Comment:
I will be changing this to `repository/ld4p` per discussion in Slack today, once I get out of the `async`/`await` woods.